fe/fe-core/src/main/java/com/starrocks/alter/Alter.java

```diff
@@ -374,6 +377,17 @@ public void processAlterTable(AlterTableStmt stmt) throws UserException {
         // some operations will take long time to process, need to be done outside the databse lock
         boolean needProcessOutsideDatabaseLock = false;
         String tableName = dbTableName.getTbl();
+
```

Comment:
1. What is the behavior at the current time?
2. What's the behavior of MySQL?
You have to describe the bug and solution clearly in the commit message.

Reply:
done